mints/whitelist.txt

```diff
@@ -0,0 +1,3 @@
+contract/valory/agent_registry/0.1.0
```

Comment:
Why are we not minting this?

Reply:
I basically wrote a script which skipped the registry contracts from the service repos since they already exists on the core repo, but this is a different type of registry. I've fixed it in the latest commit